### ISSUE TITLE
Audit inline styles causing CSP warnings

### ISSUE THREAD

**Issue by jaredhirsch** [mozilla/testpilot#2560]
Refs #2148

**Comment by johngruen** [comment]
grepped for `style=` in `/frontend/src/app`. Sadly, none of these seem to be easily resolvable ATM. I'm going to unassign myself until we have a full new set of mockups for this page.
```
78:        { this.state.privacyNote ? <span data-l10n-id="newsletterFormPrivacyAgreementRequired" style={{ color: 'red', marginRight: '0.5em' }}></span> : null }

containers/ExperimentPage.js
323:          <div className="sticky-header-sibling" style={{ height: `${stickyHeaderSiblingHeight}px` }} ></div>
638:            <button onClick={e => { e.preventDefault(); this.setState({ showEolDialog: true }); }} style={{ minWidth: progressButtonWidth }} id="uninstall-button" className={classnames(['button', 'warning'], { 'state-change': isDisabling })}><span className="state-change-inner"></span><span data-l10n-id="disableExperimentTransition" className="transition-text">Disabling...</span><span data-l10n-id="disableExperiment" data-l10n-args={JSON.stringify({ title })} className="default-text"></span></button>
646:        <button disabled onClick={e => { e.preventDefault(); }} style={{ minWidth: progressButtonWidth }} id="install-button"  className={classnames(['button', 'default'])}><span data-l10n-id="experimentManuallyDisabled" data-l10n-args={JSON.stringify({ title })} className="default-text"></span></button>
653:          <button onClick={e => this.renderUninstallSurvey(e)} style={{ minWidth: progressButtonWidth }} id="uninstall-button" className={classnames(['button', 'secondary'], { 'state-change': isDisabling })}><span className="state-change-inner"></span><span data-l10n-id="disableExperimentTransition" className="transition-text">Disabling...</span><span data-l10n-id="disableExperiment" data-l10n-args={JSON.stringify({ title })} className="default-text"></span></button>
660:        <button onClick={e => this.installExperiment(e)} style={{ minWidth: progressButtonWidth }} id="install-button"  className={classnames(['button', 'default'], { 'state-change': isEnabling })}><span className="state-change-inner"></span><span data-l10n-id="enableExperimentTransition" className="transition-text">Enabling...</span><span data-l10n-id="enableExperiment" data-l10n-args={JSON.stringify({ title })} className="default-text"></span></button>
```

**Comment by chuckharmston** [comment]
Some of these are still not easily-resolvable. Should we allow `unsafe-inline` styles here?

**Comment by ghost** [comment]
This will be fixed with the new mobile issues #3504
